lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -683,6 +702,53 @@ def test_grading_failure(self, error_message, mock_grades_iter, _mock_current_ta
             }
         ])
 
+    @patch('lms.djangoapps.instructor_task.tasks_helper.runner._get_current_task')
+    def test_inactive_enrollment_included(self, _get_current_task):
+        """Students with inactive enrollments in a course should be included in Problem Grade Report."""
```

Comment:
Nit: docstring isn't required and can become an unnecessary maintenance burden if the method and parameter names are well chosen and the docstring doesn't add any additional information.